Doc/Tutorial/chapter_uniprot.tex

```diff
@@ -57,16 +57,20 @@ \subsection{Parsing Swiss-Prot records}
 %TODO - Check the single quotes when printing record.description here:
 \begin{verbatim}
 >>> print(record.description)
-'RecName: Full=Chalcone synthase 3; EC=2.3.1.74; AltName: Full=Naringenin-chalcone synthase 3;'
```

Comment:
there was no `myaccessionnumber` specified before.

I've refrained to make the test online since I remember you mentioned a problem with the CI on Windows.